src/Square.java

```diff
@@ -0,0 +1,39 @@
+public class Square {
+    private int position;
+    private int delta = 0;
+    private Player player;
+
+    public int getPosition() {
+        return position;
+    }
+
+    public void setPosition(int position) {
+        this.position = position;
+    }
+
+    public int getDelta() {
+        return delta;
+    }
+
+    public void setDelta(int delta) {
+        this.delta = delta;
+    }
+
+    public Player getPlayer() {
+        return player;
+    }
+
+    public void setPlayer(Player player) {
+        this.player = player;
+    }
+
+    @Override
+    public String toString() {
+        if(getDelta() != 0 && getPlayer()!= null)
```

Comment:
jak już używasz wcześniej `{}` to konsekwentnie używaj dalej. Brak `{}` będzie powodować problemy.
Formatowanie